aptible/resource_app_test.go

```diff
@@ -184,6 +184,7 @@ func testAccAptibleAppDeploy(handle string) string {
 		}
     service {
 			process_type = "cmd"
+			container_profile = "m4"
```

Comment:
might be easy to add a validation test that passes in an invalid container profile, i've seen similar unit tests as a guard

i think we need to be mindful of this if we support other container profiles, right? we'll have to append to this list. is there an endpoint where we can retrieve that information in case this gets stale?

Reply:
We don't currently but I've been thinking the same thing about everything I've added `StringInSlice` validations to. The others make a little more sense, e.g. `aptible_metric_drain.drain_type`, as the types determine what attributes are necessary and not allowed. In this case, I could see issuing a warning like:

> container_profile "a5" is not known to be a supported profile. Known profiles are [m4 c5 r5]. Application of this resource may fail. You may need to update to a newer version of the aptible provider.

This would still allow us to do frontend checking but wouldn't stop them from using a new container profile we add down the line. What do you think about that?

Reply:
I think that's a good start using the contextual alerts/warnings, but that might add some scope to edit those functional signatures on CRU (probably just the U?) of apps/dbs